fix for additinal easyauth settings (#1114)
MicrosoftDocs/entra-docs: docs/identity-platform/custom-extension-tokenissuancestart-configuration.md

1. Select **Workforce** as the tenant type.
1. Under **App registration** select **Pick an existing app registration in this directory** for the **App registration type**, and pick the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
1. Enter the following issuer URL, `https://login.microsoftonline.com/{tenantId}/v2.0`, where `{tenantId}` is the tenant ID of your workforce tenant.
1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.

> 1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter tenant id in GUID format.

Reviewer: ```suggestion
1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your workforce tenant ID.
```

> 1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
> 1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your workforce tenant ID.

Reviewer 2: ```suggestion
1. Under **Client application requirement**, select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
1. Under **Tenant requirement**, select **Allow requests from specific tenants** and enter your workforce tenant ID.
```

1. Under **Settings**, select **Authentication**.
1. Select **Add Identity provider**.  
1. Select **Microsoft** as the identity provider.
1. Select **External configuration** as the tenant type.
1. Under **App registration**, select **Provide the details of an existing app registration** for the **App registration type**, and enter the `client_id` of the *Azure Functions authentication events API* app registration you [previously created](#step-1-register-a-custom-authentication-extension) when registering the custom claims provider.
1. For the **Issuer URL**, enter the following URL `https://{domainName}.ciamlogin.com/{tenant_id}/v2.0`, where
    - `{domainName}` is the domain name of your external tenant, in the form `{domainName}.contoso.com`.
    - `{tenantId}` is the tenant ID of your external tenant.

> 1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
> 1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your external tenant ID.

Reviewer 2: ```suggestion
1. Under **Client application requirement**, select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
1. Under **Tenant requirement**, select **Allow requests from specific tenants** and enter your external tenant ID.
```

1. For the **Issuer URL**, enter the following URL `https://{domainName}.ciamlogin.com/{tenant_id}/v2.0`, where
    - `{domainName}` is the domain name of your external tenant, in the form `{domainName}.contoso.com`.
    - `{tenantId}` is the tenant ID of your external tenant.

> 1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.

Reviewer: Also line L390 - Select **External configuration** as the tenant for your application

Author: fixed. also changed L391

> 1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter tenant id in GUID format.

Reviewer: ```suggestion
1. Under **Tenant requirement** select **Allow requests from specific tenants** and enter your external tenant ID.
```
